app/src/main/java/xterminators/spellingbee/cli/CLIController.java

```diff
@@ -363,14 +420,115 @@ private void ranks() {
+            //Returns true if a new file is created.
+                if(savedFile.createNewFile()){
+
+                    //Create a file writer to populate the created File
+                    FileWriter writing = new FileWriter(savedFile);
+                    //Insert input
+                    writing.write (savedJson);
+                    //Close the writer
+                    writing.close ();
+                    //Notify the user
+                    System.out.println("File created: " + filename + ".json");
+                }
+            
+        } catch (IOException e) {
+            System.out.println("An error occurred");
+            }
+        
     }
 
     /**
      * Saves the puzzle to a json file at a default location.
      */
     private void save() {
-        // TODO: Implement default save
+        //Create an object of the Gson class
+        Gson saved = new Gson();
+
+        char[] nonRequiredLetters = puzzle.getSecondaryLetters();
+
+        char[] baseWord = new char[7];
+        baseWord[6] = puzzle.getPrimaryLetter();
+        for(int i = 0; i < baseWord.length - 1; i++){
+            baseWord[i] = nonRequiredLetters[i];
+        }
+
+        String filename = "";
+
+        //This will create a title for the Json file consisting
+        // of the non-required letters followed by the required letter.
+        for (char c : baseWord){
+            filename = filename + c;
+        }
+
+        
+
+        //Create the file and populate it with the saved Json
+        try{
+            // Take the necessary attributes and create a puzzleSave object,
+            PuzzleSave savedPuzzle = PuzzleSave.ToSave(baseWord, puzzle.getFoundWords(), puzzle.getEarnedPoints(), puzzle.getPrimaryLetter(), puzzle.getTotalPoints());
+
+            //Converts the current puzzle object to Json
+            String savedJson = saved.toJson (savedPuzzle);
+
+            File savedFile = new File(filename + ".json");
+
+            //Returns true if a new file is created.
+            if(savedFile.createNewFile ()){
+
+                //Create a file writer to populate the created File
+                FileWriter writing = new FileWriter(savedFile);
+                //Insert input
+                writing.write (savedJson);
+                //Close the writer
+                writing.close ();
+                //Notify the user
+                    System.out.println("File created: " + filename + ".json");
+            } else {
+                System.out.println("A file by that name already exists." + getNewLineCharacter() + "Overwriting the file");
+
+                //Open a writer to replace the information in the file.
+                PrintWriter writer = new PrintWriter(savedFile);
+                writer.print(savedJson);
+                writer.close();
+            }
+        }
+        catch (IOException e) {
+            System.out.println("An error occurred");
+        }
```

Comment:
This should probably just be a call to save(String) with your built default file name.